### ISSUE TITLE
REQ: New(old) mode: berzerk

### ISSUE THREAD

**Issue by blaps** [deurk/ktx#123]
> This feature request has been lifted from /ezquake to here.

Almost forgotten is the old [KTpro](https://www.quakeworld.nu/wiki/KTX) command `/berzerk`. Now it would be awesome to have it resurface.

Back then it could be set by one player, but should probably be votable.
When enabled, `berzerk` activates when there is 20 seconds left of a match. The following happens:
- Everyone gets quad for the remaing time (even if you die and respawn)
- Screen flickers with an epileptic strobe effect once every second
Classic mayhem!

Vague recollection: did it also disable LG? Potential bore-fest. Also, it should display in countdown centerprint when enabled.

**Comment by deurk** [comment]
lol

**Comment by blaps** [comment]
Don't you dare lol at me! ;D

**Comment by blaps** [comment]
This is how public demand looks like down on the street
![image](https://user-images.githubusercontent.com/8357443/94659233-3d695d80-0304-11eb-9458-f5ca0fd35156.png)

**Comment by deurk** [comment]
Funny. it's part of the ktx configuration but nothing appears in the code...
```
set k_bzk                     0         // berzerk (0 = off, 1 = on)
set k_btime                   20        // berzerk time (seconds)
```

**Comment by tcsabina** [comment]
I found the ktpro v2.20 source code: https://drive.google.com/file/d/1YBvC5-VSuFTWaMBLFO45B0fUZ8epIiUs/view?usp=sharing

as a starting point...

**Comment by deurk** [comment]
The code you are sharing is not open source and should not be distributed without prior author consent!

**Comment by tcsabina** [comment]
oh. okay.
what if there is no way to contact them?

**Comment by deurk** [comment]
then it's down the tricky part...

I guess interested people will know you can provide it to serve as a reference if anyone is interesting into giving it a go :)

**Comment by tcsabina** [comment]
Implemented with PR #144. Please merge.

**Comment by tcsabina** [comment]
Available from KTX 1.40
